bindings/java/src/lib.rs

```diff
@@ -15,20 +15,29 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use std::cell::RefCell;
 use std::collections::HashMap;
 use std::str::FromStr;
+use std::sync::Arc;
 
-use jni::objects::JClass;
 use jni::objects::JMap;
 use jni::objects::JObject;
 use jni::objects::JString;
+use jni::objects::{JClass, JValue};
 use jni::sys::jboolean;
 use jni::sys::jlong;
-use jni::JNIEnv;
+use jni::{JNIEnv, JavaVM};
+use tokio::runtime::{Builder, Runtime};
+
 use opendal::BlockingOperator;
 use opendal::Operator;
 use opendal::Scheme;
 
+thread_local! {
```

Comment:
Maybe we can refer to https://github.com/jni-rs/jni-rs/blob/master/example/mylib/src/lib.rs#L145?

Reply:
`attach_current_thread` is a heavy operation, it is not a good choice to attach JVM in every operation.

Reply:
> As attachment and detachment of a thread is an expensive operation, the scoped attachment shall be used if happens infrequently.

Reference: https://docs.rs/jni/latest/jni/struct.JavaVM.html#attaching-native-threads